package/docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,28 @@
+ARG BASE_IMAGE=bionic
+FROM ubuntu:${BASE_IMAGE}
+ARG BASE_IMAGE=bionic
```

Comment:
Why is `ARG BASE_IMAGE=bionic` here twice?

Reply:
In a `Dockerfile`, anytime there is a `FROM` clause, all `ARG` are reset. So you need the one before, and the one after, to make sure that `BASE_IMAGE` is available both for the `FROM` clause and after the `FROM` clause.